### ISSUE TITLE
Leaderboards 'Rows per Table' select is not accessible

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/wc-admin#1644]
**To Reproduce**
With a screen reader go to the _Dashboard_.

1. Open the _Leaderboards_ menu.
2. Navigate through it with the tab key.
3. Notice the screen reader doesn't read _Rows Per Table_, so when reaching the select box the user doesn't know what it refers to.

**Expected behavior**
The screen reader should read the _Rows Per Table_ title and, even better, it should be the label of select element.

**Screenshots**
![image](https://user-images.githubusercontent.com/3616980/53089836-f2734b80-350d-11e9-91c2-b5491db11756.png)
